src/VisualStudio/CSharp/Impl/Options/IntelliSenseOptionPageControl.xaml

```diff
@@ -80,6 +80,14 @@
                               Unchecked="Show_items_from_unimported_namespaces_CheckedChanged"
                               Indeterminate="Show_items_from_unimported_namespaces_CheckedChanged"
                               IsThreeState="True"/>
+
+                    <CheckBox x:Uid="Automatically_add_parenthesis_by_semicolon"
```

Comment:
My understanding is we don't need an option for this, since user's intention is clear when ";" is typed. We only need an option to determine the behavior of "Tab".

Reply:
Removed